tsconfig.json

```diff
@@ -9,7 +9,7 @@
           },
         "target": "ES2020",
         "useDefineForClassFields": true,
-        "lib": ["ES2020", "DOM", "DOM.Iterable"],
+        "lib": ["ES2020", "ES2023.Array", "DOM", "DOM.Iterable"],
```

Comment:
Unlike es2020, es2023 supports `findLastIndex()` array method, which I'm using and its working wee but our current TS linter doesn't recognize it without this setting.